bin/browsertime.js

```diff
@@ -94,6 +101,44 @@ async function run(urls, options) {
 
 let cliResult = cli.parseCommandLine();
 
+/*
+  Each url can be:
+   - an url value
+   - an array of tests. In that case it's a mapping containing
+     theses values:
+     - test: an async function containing the test to run
+     - setUp: an async function for the preScript [optional]
+     - tearDown: an async function for the postScript [optional]
+*/
+var tests = [];
```

Comment:
const is the new var :) Meaning in our code you can almost always use const (or let). I should document that too, sorry.